src/components/common/Dropdown.jsx

```diff
@@ -3,31 +3,51 @@ import React from 'react';
 const Dropdown = () => (
   <div className="dropdown is-active">
     <div className="dropdown-trigger">
-      <button className="button" aria-haspopup="true" aria-controls="dropdown-menu">
+      <button className="button" type="button" aria-haspopup="true" aria-controls="dropdown-menu">
         <span>Dropdown button</span>
         <span className="icon is-small">
-          <i className="fas fa-angle-down" aria-hidden="true"></i>
+          <i className="fas fa-angle-down" aria-hidden="true" />
         </span>
       </button>
     </div>
     <div className="dropdown-menu" id="dropdown-menu" role="menu">
       <div className="dropdown-content">
-        <a href="#" className="dropdown-item">
+        <button
```

Comment:
This was probably the most strange change but seeing as how this isnt being used yet im sure we can figure out a lint-safe way to do this

Reply:
pull latest dev, this has been fixed